marketing.go

```diff
@@ -1,13 +1,23 @@
 package hubspot
 
+import "fmt"
+
 const (
 	marketingBasePath = "marketing"
 )
 
 type Marketing struct {
-	Email MarketingEmailService
+	Email         MarketingEmailService
+	Transactional TransactionalServiceOp
 }
 
+// "/marketing/v3/transactional/single-email/send"
```

Comment:
GoDoc to explain this function is supposed  to come here. How about this comment to move to the above of definition  `fmt.Sprintf("%s/%s/%s", marketingBasePath, c.apiVersion, transactionalBasePath)`?

Reply:
This was unintended, removed.